src/layout/bubbles.svelte

```diff
@@ -25,6 +25,11 @@
     { left: 92, size: 108, delay: 0, duration: 8, popping: false }
   ]
 
+  if (window.location.pathname === '/bubbles') {
+    const times = prompt('Please enter number of bubbles (10 - 200):', 10)
```

Comment:
I would prefer passing this parameter in URL. Like /bubbles/50.

Reply:
changed.